dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -5517,6 +5518,28 @@ fn inv_collection_print_sleds(collection: &Collection) {
     }
 }
 
+fn inv_collection_print_keeper_membership(collection: &Collection) {
+    println!("\nKEEPER MEMBERSHIP");
+    for k in &collection.clickhouse_keeper_cluster_membership {
+        println!("\n    queried keeper: {}", k.queried_keeper);
+        println!(
+            "    leader_committed_log_index: {}",
+            k.leader_committed_log_index
+        );
+        let raft_config =
+            k.raft_config.iter().fold("".to_string(), |mut acc, id| {
+                acc.push_str(&format!("{id}, "));
+                acc
+            });
+        let s = raft_config.trim_end_matches(", ");
```

Comment:
Maybe worth using [`itertools`](https://docs.rs/itertools/latest/itertools/trait.Itertools.html#method.join) here? I think it'd let this block be

```rust
let s = k.raft_config.iter().join(", ");
```

Reply:
Wow, that's much simpler. TIL. Thanks!

Done in [94904b1](https://github.com/oxidecomputer/omicron/pull/6950/commits/94904b1b5e11212c65ff3fd41d2f35e5df3759b8)